README.md

```diff
@@ -238,3 +238,5 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [Code-Server](/software/code-server)
 ### haste-server
 * [haste-server](/software/haste-server)
+### Grafana
+* [Grafana](/software/grafana)
```

Comment:
Please keep the alphabetical order with grafana being above haste

Reply:
Fixed